--- nml-0.3.1.r5242-f6a3ae1163ab/nml/actions/action5.py.orig	2015-05-12 10:24:15.352013046 +0100
+++ nml-0.3.1.r5242-f6a3ae1163ab/nml/actions/action5.py	2015-05-12 10:24:31.493013672 +0100
@@ -78,6 +78,7 @@
     'OTTD_GUI' : (0x15, 175, Action5BlockType.OFFSET),
     'AIRPORT_PREVIEW' : (0x16, 9, Action5BlockType.OFFSET),
     'RAILTYPE_TUNNELS': (0x17, 16, Action5BlockType.OFFSET),
+    'OTTD_RECOLOUR' : (0x18, 1, Action5BlockType.OFFSET),
 }
 
 def parse_action5(replaces):
